src/modules/CatalogPage/CatalogPage.module.scss

```diff
@@ -0,0 +1,112 @@
+@import '../../styles/blocks/container';
+@import '../../styles/colors';
+@import '../../styles/blocks/grid';
+
+.title {
+  display: block;
+  font-weight: 800;
+  font-size: 32px;
+  margin-bottom: 8px;
+
+  @include onTablet {
+    font-size: 48px;
+  }
+}
+
+.breadcrumbs {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  margin-bottom: 24px;
+
+  @include onTablet {
+    margin-bottom: 40px;
+  }
+}
+
+.step {
+  font-size: 12px;
+  color: $color__secondary;
+  text-decoration: none;
+  font-weight: 600;
+}
+
+.homeIcon {
+  background-image: url(../../static/icons/Home.svg);
+  width: 16px;
+  height: 16px;
+}
+
+.iconNext {
+  background-image: url(../../static/icons/Chevron_Arrow_Right.svg);
+  width: 16px;
+  height: 16px;
+}
+
+.amount {
+  display: block;
+  color: $color__secondary;
+  font-size: 14px;
+  font-weight: 600;
+  margin-bottom: 32px;
+
+  @include onTablet {
+    margin-bottom: 40px;
+  }
+}
+
+.selects {
+  display: flex;
+  margin-bottom: 24px;
+  gap: 16px;
+}
+
+.selectWrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+.sort {
+  width: 136px;
+  height: 40px;
+  border-radius: 8px;
+  border: 1px solid $color__icons;
+  background-color: $color__hover-and-bg;
+  padding-inline: 12px;
+  appearance: none;
+  background-image: url(../../static/icons/Chevron_Arrow_Down.svg);
+  background-repeat: no-repeat;
+  background-position: right 1em top 50%;
```

Comment:
You can put all background values in one:
background: url(../../static/icons/Chevron_Arrow_Down.svg) no-repeat right 1em top 50%;